monologue/src/main/java/monologue/EvalMethod.java

```
@@ -16,14 +16,20 @@ private static Supplier<?> getSupplier(Method method, Logged loggable) {
       try {
         return method.invoke(loggable);
       } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
-        DriverStation.reportWarning(
-            method.getName() + " supllier is erroring: " + e.toString(), false);
+        MonologueLog.runtimeWarn(

```

In my experience this warning is just console spam with no guide for what to fix, but IDK how to improve the message.

---

I've personally never had it happen, I would have assumed the `e.toString` provides insight to the error. Silently failing IMO is worse and it does provide a path to what is causing a problem.